src/ZkFold/Cardano/UPLC/Rollup.hs

```diff
@@ -81,14 +80,18 @@ rollup (RollupSetup ledgerRules dataCurrency threadValue feeAddress) (UpdateRoll
     -- Get bridge outputs
     -- If the payment credential of the output coincides with the rollup payment credential, then this output transfers value to the rollup.
     -- Otherwise, it transfers value from the rollup.
-    bridgeOutputs =
-      filter (\case
-        TxOut _ _ (OutputDatumHash _) Nothing -> True
-        _                                     -> False)
-      $ tail $ tail $ tail $ txInfoOutputs $ scriptContextTxInfo ctx
+    -- bridgeOutputs =
```

Comment:
Temporarily disabling `bridgeOutputs` for testing purposes.  It seems that we want the "change output" to appear just before the bridge outputs in the `TxInfoOutputs` list.  So next step with regard to testing is to verify that I can control this order in the cardano-cli offchain code (must be possible).